pkg/liquidity-mining/contracts/fee-distribution/FeeDistributor.sol

```diff
@@ -450,7 +452,12 @@ contract FeeDistributor is IFeeDistributor, ReentrancyGuard {
      */
     function _getUserTokenTimeCursor(address user, IERC20 token) internal view returns (uint256) {
         uint256 userTimeCursor = _userTokenTimeCursor[user][token];
-        return userTimeCursor > 0 ? userTimeCursor : _tokenStartTime[token];
+        if (userTimeCursor > 0) return userTimeCursor;
```

Comment:
Not quite related to this PR, but I find the existence of this function a bit strange as it is only used in `_claimToken`. All other functions direcfly access the mapping, including `getUserTokenTimeCursor` (!).